python/cuml/manifold/t_sne.pyx

```diff
@@ -169,6 +171,9 @@ class TSNE(Base):
     specifying random_state and fixing it across runs can help, but TSNE does
     not guarantee similar results each time.
 
+    As suggested, PCA (in future release) can also help to alleviate this
```

Comment:
Maybe this could be a little clearer? Something like "PCA initialization (upcoming with change #<insert-pr-num-here>) can alleviate this.

Reply:
Oh yep good idea!